docs/usage/registry-cache/configuration.md

```diff
@@ -49,6 +49,7 @@ spec:
       kind: RegistryConfig
       caches:
       - upstream: docker.io
+        remoteURL: https://registry-1.docker.io
```

Comment:
Cosmetic nit: I would drop the `upstreamURL` field for this cache to avoid confusion as below we already have an example of remoteURL usage.